doc/src/suite-config.rst

```diff
@@ -169,17 +169,13 @@ Jinja2 (:ref:`Jinja`) also has template inclusion functionality.
 Syntax Highlighting For Suite Configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Cylc comes with syntax files for a number of text editors:
-
-.. code-block:: bash
-
-   <cylc-dir>/etc/syntax/cylc.vim     # vim
-   <cylc-dir>/etc/syntax/cylc-mode.el # emacs
-   <cylc-dir>/etc/syntax/cylc.lang    # gedit (and other gtksourceview programs)
-   <cylc-dir>/etc/syntax/cylc.xml     # kate
+Cylc comes with syntax files for a number of text editors: vim, emacs,
+gedit (and other gtksourceview programs), and kate.
 
 Refer to comments at the top of each file to see how to use them.
 
+.. TODO: TELL USERS WHERE TO GET THIS FROM.
```

Comment:
I'm not sure this would be possible under the new system, but I thought I'd raise it here for future consideration: ideally users would symlink these files (also the bash CLI completion) so they get updated, otherwise they will get quickly outdated.

Conda might possibly make this sort of thing easier by allowing us to install non-python files into a Conda environment but I don't know that editors would be able to handle this.

Perhaps as a halfway solution we could provide automatic (user) installation to reduce the barrier to installation/upgrading.

Reply:
For the moment I'll replace this TODO with instructions to use the new `cylc get-pkg-resources` command, and I have updated #3150 on this topic.